src/index.ts

```diff
@@ -16,11 +16,11 @@ const nexusSdk = {
   productTypes: productTypesData,
   ...generatedTypes,
   ...swap,
-  ...buyCover,
   ...types,
   ...quote,
   ...ipfs,
   ...constants,
```

Comment:
since Swap, Quote and IPFS classes are under NexusSDK class maybe we should not expose them?